tests/linker/ios/link sdk/LinkSdkRegressionTest.cs

```diff
@@ -1043,14 +1043,16 @@ public void TlsProvider_Apple ()
 		public void Github5024 ()
 		{
 			TestRuntime.AssertXcodeVersion (6,0);
-			var sc = new UISearchController ((UIViewController) null);
-			sc.SetSearchResultsUpdater ((vc) => { });
-
-			var a = typeof (UISearchController).AssemblyQualifiedName;
-			var n = a.Replace ("UIKit.UISearchController", "UIKit.UISearchController+__Xamarin_UISearchResultsUpdating");
-			var t = Type.GetType (n);
-			Assert.NotNull (t, "private inner type");
-			Assert.IsNotNull (t.GetMethod ("UpdateSearchResultsForSearchController"), "preserved");
+			using (var controller = new UISplitViewController ()) {
+				var sc = new UISearchController ((UIViewController)controller);
+				sc.SetSearchResultsUpdater ((vc) => { });
```

Comment:
I know this is not introduced here, but what does this syntax mean? 
```(vc) => { }```

Reply:
If you look at the method, the first param is a Action<UIKit.UISearchController> updateSearchResults. The test is simply adding a lambda that takes a first param (viewController) and does nothing.

Reply:
Thanks for explaining! :)